Sources/Card/GetOrderInfoRequest.swift

```diff
@@ -8,15 +8,14 @@ struct GetOrderInfoRequest: APIRequest {
     typealias ResponseType = GetOrderInfoResponse
 
     let clientID: String
-    let secret: String
     let pathFormat: String = "v2/checkout/orders/%@"
 
     var path: String
     var method: HTTPMethod = .get
     var body: Data?
 
     var headers: [HTTPHeader: String] {
-        let encodedClientID = "\(clientID):\(secret)".data(using: .utf8)?.base64EncodedString() ?? ""
+        let encodedClientID = "\(clientID)".data(using: .utf8)?.base64EncodedString() ?? ""
```

Comment:
We may need to use an access token here. I'm not sure if Basic auth with `clientID` only will work for this endpoint.